__mocks__/@react-navigation/native/index.ts

```diff
@@ -1,7 +1,7 @@
 import {useIsFocused as realUseIsFocused} from '@react-navigation/native';
 
 // We only want this mocked for storybook, not jest
-const useIsFocused = process.env.NODE_ENV === 'test' ? realUseIsFocused : () => true;
+const useIsFocused: () => boolean = process.env.NODE_ENV === 'test' ? realUseIsFocused : () => true;
```

Comment:
Maybe it makes sense to type it this way? What do you think?
```suggestion
const useIsFocused: typeof realUseIsFocused = process.env.NODE_ENV === 'test' ? realUseIsFocused : () => true;
```

Reply:
yeah I think it will be better!